tools/internal/ios/core-test-runner/core-test-runner.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,8 +5,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/iTwin/mobile-native-ios",
       "state" : {
-        "revision" : "b365b0ca9fe2b16cddaf26f59e8f38148027608c",
-        "version" : "4.8.11"
+        "revision" : "c011577eed1b76b408a7d997be918b41380fb836",
```

Comment:
curious, where does this hash come from? we have scripts to update addon version across the monorepo, should we automate this too, or rather could we?

Reply:
The hash is normally populated by Xcode itself when it updates this file (as my Xcode did). I'm not entirely sure where it comes from. Having said that, 4.8.11 was wrong for the master branch, so I included this change in my PR.